tests/confreq-rs-xml-nval_comment/OPS/package.opf

```diff
@@ -25,7 +25,7 @@
 				    
   
 
-<itemref idref="content_001" />
+<itemref idref="nav" />
```

Comment:
Can you fix the indentation/whitespace around the comment, unless it's an important part of the XML parsing test? I want it to be clear what parts are actually relevant to the test. (Today it looks to me like we think XML parsers might be expecting consistent whitespace, or handling linebreaks badly, which I assume is unlikely but I don't really know.) In short, clean markup when possible. I have fixed indentation throughout all existing tests.

Reply:
This was deliberately part of the parsing test. XML comments out in the world are often messy. This would likely break a GREP-based parser, which is a spectacularly bad idea :)